components/Common/Search/States/WithSearchBox.tsx

```diff
@@ -31,6 +32,7 @@ type SearchBoxProps = { onClose: () => void };
 export const WithSearchBox: FC<SearchBoxProps> = ({ onClose }) => {
   const [searchTerm, setSearchTerm] = useState('');
   const [searchResults, setSearchResults] = useState<SearchResults>(null);
+  const [selectedResult, setSelectedResult] = useState<Nullable<number>>();
```

Comment:
This doesn't need to be a nullable, if you set "null" as the default value. TypeScript will automatically infer that. At the moment this type is "null | undefined | number" if you do `useState<number>(null)` it'll be automatically `null | number`

Reply:
Does it need to be null? Just out of curiosity... Why not undefined?

Reply:
No reason actually, changed it to `number|undefined` 👍🏻